cargo-dist/src/errors.rs

```diff
@@ -599,18 +609,6 @@ pub enum DistError {
         /// Version the project uses
         your_version: semver::Version,
     },
-
-    /// axoupdater can't be built for this target from this host
-    #[error("You've requested a cross-compile from {host} to {target}, but axoupdater can't be built for this target")]
-    #[diagnostic(help(
-        "Please either disable this target, or set `install-updater' to `false' in your config."
```

Comment:
this isn't really a comment for this PR but i would expect that folks would like the opportunity to turn off the updater for certain targets

Reply:
True, that could be a useful feature in the future!